systemtest/src/test/java/io/strimzi/systemtest/report/DefaultClusterReportST.java

```diff
@@ -0,0 +1,448 @@
+            "--cluster=" + testStorage.getClusterName(),
+            "--out-dir=" + outPath,
+            "--bridge=my-bridge",
+            "--connect=my-connect",
+            "--mm2=my-mm2"
+        );
+
+        assertThat("Output directory does not exist", FileUtils.exists(outPath));
+        assertThat("Output ZIP file does not exist", FileUtils.listFilesWithSuffix(outPath, ".zip").length == 1);
+
+        assertValidClusterRoleBindings(outPath);
+        assertValidClusterRoles(outPath);
+        assertValidConfigMaps(outPath, testStorage.getClusterName());
+        assertValidCustomResourceDefinitions(outPath);
+        assertValidCustomDeployments(outPath, testStorage.getClusterName());
+        assertValidNetworkPolicies(outPath, testStorage.getClusterName());
+        assertValidPodDisruptionBudgets(outPath, testStorage.getClusterName());
+        assertValidPods(outPath, testStorage.getClusterName());
+        assertValidReplicaSets(outPath, testStorage.getClusterName());
+        assertValidRoleBindings(outPath, testStorage.getClusterName());
+        assertValidRoles(outPath, testStorage.getClusterName());
+        assertValidSecrets(outPath, testStorage.getClusterName());
+        assertValidServices(outPath, testStorage.getClusterName());
+
+        assertValidKafkaBridges(outPath);
+        assertValidKafkaConnects(outPath);
+        assertValidKafkaMirrorMaker2s(outPath);
+        assertValidKafkaRebalances(outPath, testStorage.getClusterName());
+        assertValidKafkas(outPath, testStorage.getClusterName());
+        assertValidKafkaTopics(outPath);
+        assertValidKafkaUsers(outPath);
+        assertValidStrimziPodSets(outPath, testStorage.getClusterName());
+
+        assertValidConfigs(outPath, testStorage.getClusterName());
+        assertValidLogs(outPath, testStorage.getClusterName());
+
+        Secret clusterCaSecret = getSecretWithKeyFromFile(secretPath, secretKey);
+        assertThat("Keys are not hidden in secrets", clusterCaSecret.getData().get(secretKey).equals("<hidden>"));
+
+        assertThat("KafkaBridge CRD does not exist", FileUtils.exists(outPath + "/reports/customresourcedefinitions/kafkabridges.kafka.strimzi.io.yaml"));
+        assertThat("KafkaBridge logs are missing", FileUtils.listFilesWithPrefix(outPath + "/reports/logs", "my-bridge-bridge").length == 1);
+
+        assertThat("KafkaConnect CRD does not exist", FileUtils.exists(outPath + "/reports/customresourcedefinitions/kafkaconnects.kafka.strimzi.io.yaml"));
+        assertThat("KafkaConnect logs are missing", FileUtils.listFilesWithPrefix(outPath + "/reports/logs", "my-connect-connect").length == 1);
+
+        assertThat("KafkaMirrorMaker2 CRD does not exist", FileUtils.exists(outPath + "/reports/customresourcedefinitions/kafkamirrormaker2s.kafka.strimzi.io.yaml"));
+        assertThat("KafkaMirrorMaker2 logs are missing", FileUtils.listFilesWithPrefix(outPath + "/reports/logs", "my-mm2-mirrormaker2").length == 1);
```

Comment:
out of curiosity, why checking the "main" CRD so `Kafka` is missing here?

Reply:
These are actually a refusal from my previous implementation, as they are already covered by previous validate methods. Let me remove them.